### ISSUE TITLE
Seems to be a small bug in PageButton.js

### ISSUE THREAD

**Issue by divye01** [AllenFang/react-bootstrap-table#1189]
When the list size is 1, it seems that the "Next" button is enabled but hidden. So the page-item property for the "next" button seems to be 'true': 

```
<ul class="react-bootstrap-table-page-btns-ul pagination">
<li class="active page-item"><a href="#" class="page-link">1</a>
</li><li class="disabled hidden page-item"><a href="#" class="page-link">Next</a></li></ul>
```

**Comment by AllenFang** [comment]
could you please show me your configuration, and does any critical bug here? thanks

**Comment by divye01** [comment]
So I am using Remote loading in the table. There are two minor bugs I found out, nothing critical: 

1. When there is only one page in the table, the top right round corner, bottom right round corner property is not applied to the pagination. Since it is not the last child, (the last child is hidden)
![screen shot 2017-04-03 at 11 45 10 am](https://cloud.githubusercontent.com/assets/22155632/24617905/4e9c4296-1863-11e7-89ad-17f5e91e3633.png)

2. When you are not on the first page, and search for an item, the list size reduces dynamically. There is a Next button displayed even when there's not another page. It disappears when you click it. 
![screen shot 2017-04-03 at 11 48 22 am](https://cloud.githubusercontent.com/assets/22155632/24618009/a0bc9b16-1863-11e7-93d1-d12446bdd1be.png)

`
const options = {
          page: 1,
         pageStartIndex: 1,
         paginationSize: 3,
         prePage: 'Prev',
         nextPage: 'Next',
         firstPage: 'First',
         lastPage: 'Last',
        sizePerPageList: [
          {text: '10', value: 10},
          {text: '20', value: 20},
          {text: '50', value: 50},
          {text: 'All', value: this.props.listSize}
        ],
        fetchInfo: {
          dataTotalSize:  this.props.listSize
        },
        onSizePerPageList: this.props.changeSubstanceListSizePerPage.bind(this),
        onPageChange: this.props.changeSubstanceListCurrentPage.bind(this),
        onSortChange: this.props.sortSubstanceList.bind(this),
        sizePerPage: this.props.listSizePerPage,
        page: this.props.currentPage
      };

<BootstrapTable
                  striped
                  hover
                  condensed
                  pagination
                  data={this.props.listData}
                  options={options}
                  remote={true}
                  fetchInfo={{dataTotalSize: parseInt(this.props.listSize)}}
                >
                  <TableHeaderColumn width='40px' dataField='substance_id' isKey hidden>ID</TableHeaderColumn>
                  <TableHeaderColumn dataSort={true} dataFormat={listFormatter} dataField='common_name'>{copy.commonName[lang]}</TableHeaderColumn>
                  <TableHeaderColumn filterFormatted dataFormat={listFormatter} dataSort={true} dataField='proper_name'>{copy.properName[lang]}</TableHeaderColumn>
                  <TableHeaderColumn dataSort={true} dataFormat={listFormatter} dataField='source_material'>{copy.sourceMaterial[lang]}</TableHeaderColumn>
                  <TableHeaderColumn width='100' dataField='active' dataFormat={this.viewColumnFormatter}></TableHeaderColumn>
                </BootstrapTable>
`

**Comment by AllenFang** [comment]
Thanks :) it's much better to understand after I saw your screenshot, I'll try to figure out why, thank you 👍

**Comment by divye01** [comment]
UPDATE: I just updated to the version 3.1.6 (Earlier i was using 3.0.0-beta.7). So the first bug seems to be fixed in this version. Here is the update screenshot: 

![screen shot 2017-04-03 at 1 53 04 pm](https://cloud.githubusercontent.com/assets/22155632/24623018/150632fa-1875-11e7-8d71-33d60578b4f1.png)

However the second bug is still there.

**Comment by AllenFang** [comment]
Fixed on `v3.1.8`
